src/firecracker/build.rs

```diff
@@ -19,6 +19,8 @@ fn main() {
             env!("CARGO_PKG_VERSION")
         ),
     };
+
+    Command::new("touch").args(&["build.rs"]).output().unwrap();
```

Comment:
This basically `touch`es itself as part of running, right? Does cargo consistently pick up the timestamp as _newer than last build_ even though _last build_ is basically still in progress?

If this is reliable, then fine. Otherwise we can put this `touch` in `tools/devtool` _before_ running the `cargo build [--release]` command.

Reply:
As we discussed offline, this should be reliable since cargo should cache the timestamp when the script started executing, hence this touch would be newer than last build.
And it works as expected

Reply:
If I understand correctly this is a workaround for triggering the `build.rs` every time `cargo build` is called, right?